plugins/ti-dmc/ti-dmc.quirk

```diff
@@ -0,0 +1,3 @@
+# TPS6598x
+[USB\VID_0451&PID_ACE1]
```

Comment:
Is the PID going to be the same for all these devices or is it going to be changed on an OEM by OEM basis?

Reply:
Urgh, you hit a nerve. They all have the same VID/PID with different firmware streams. I'm talking to TI about how to load the pubkey so we can add an extra instance ID.

Reply:
OK; good.  Besides the pubkey, is there an OEM specific string that can be pulled out for the device name?

Reply:
Alas not, all `Texas Instruments, Inc. TPS DMC Family`